### PR TITLE
Add wl hook

### PR DESCRIPTION
**Pull request by pablopolyte**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | set an event when a product is added to a wishlist, here we use hook
| Type?         |  new feature 
| BC breaks?    |  no
| Deprecations? |  no
| Fixed ticket? | Fixes http://forge.prestashop.com/browse/EMKTG-192.

### CODE REVIEW

File: blockwishlist.php [hunk 1]
```diff
@@ -249,4 +250,16 @@ public function hookDisplayHeader(array $params)
...
+     * This hook return idproduct and idProductAttribute when set into a wishlist
+     *
+     * @param array $params
+     */
+    public function hookActionWishlistAddProducts(array $params)
+    {
+        if (!empty($params)) {
+            Media::addJsDef($params);
+        }
+    }
```
Comment (Quetzacoalt91) [inline blockwishlist.php:264]: Why do you need to run this? What we are supposed to do with this module is to trigger other modules, not triggering himself.

Comment (pablopolyte) [reply blockwishlist.php:264]: my bad

File: blockwishlist.php [hunk 2]
```diff
@@ -39,6 +39,7 @@ class BlockWishList extends Module
         'displayAdminCustomers',
         'displayProductAdditionalInfo',
         'displayMyAccountBlock',
+        'actionWishlistAddProducts',
```
Comment (Quetzacoalt91) [inline blockwishlist.php:42]: Same comment here. You do not need to register to `ActionWishlistAddProducts` hook. You're only triggering it.

Comment (pablopolyte) [reply blockwishlist.php:42]: In devdoc they say : Every hook you want to use must be registered first.

Comment (Quetzacoalt91) [reply blockwishlist.php:42]: That's true when you need to listen to a hook, but that's not needed when you are triggering them.

File: controllers/front/action.php [hunk 3]
```diff
@@ -99,6 +99,13 @@ private function addProductToWishListAction($params)
 
         // TODO: add product to stats
 
+        Hook::exec('ActionWishlistAddProducts', [
```
Comment (zalexki) [inline controllers/front/action.php:102]:
```suggestion
        Hook::exec('ActionWishlistAddProduct', [
```

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Pull request merged**